src/vs/workbench/contrib/customEditor/browser/customEditorInput.ts

```diff
@@ -43,12 +46,16 @@ export class CustomEditorInput extends LazilyResolvedWebviewEditorInput {
 			// If it's an untitled file we must populate the untitledDocumentData
 			const untitledString = accessor.get(IUntitledTextEditorService).getValue(resource);
 			const untitledDocumentData = untitledString ? VSBuffer.fromString(untitledString) : undefined;
+			const editorGroupService = accessor.get(IEditorGroupsService);
+			const windowId = group ? editorGroupService.getGroup(group)?.windowId : undefined;
```

Comment:
I wonder if this is the right assumption. If the `group` is `undefined`, is it really always going to be `mainWindow`?